client/src/main/java/org/eclipse/hono/client/HonoClient.java

```diff
@@ -338,6 +338,49 @@
     Future<MessageConsumer> createEventConsumer(String tenantId, BiConsumer<ProtonDelivery, Message> eventConsumer,
             Handler<Void> closeHandler);
 
+    Future<MessageConsumer> createCommandResponseConsumer(String tenantId, String replyId, Consumer<Message> commandResponseConsumer,
+            Handler<Void> closeHandler);
+
+    Future<MessageConsumer> createCommandResponseConsumer(String tenantId, String replyId,
```

Comment:
this should be removed, shouldn't it?

Reply:
ups, so sorry. Something went terribly wrong with my local merging.